4-loaddatafromfirebase/firebase_counter/lib/src/todo_list/todo_list_service.dart

```diff
@@ -1,6 +1,9 @@
+// Copyright (c) 2017, janamou. All rights reserved. Use of this source code
+// is governed by a BSD-style license that can be found in the LICENSE file.
+
 import 'dart:async';
 
-import 'package:angular2/core.dart';
+import 'package:angular/core.dart';
```

Comment:
This should import `angular/angular.dart` (GLOBAL).

All angular/* imports should change to `angular/angular.dart`. (No core, no common, no browser, ...)

Reply:
@kwalrath ok, I will fix this but I created a new project in webstorm so the template should be updated as well ;-)